CHANGELOG.md

```diff
@@ -2,6 +2,177 @@
+    "data": {
+        "topProducts": [
+            {"name": "Table", "inStock": true},
+            {"name": "Chair", "inStock": false}
+        ]
+    }
+}
+```
+
+Prior to this change, if the `inventory` subgraph returns a malformed response, the router response would look like:
+```json
+{
+    "data": {"topProducts": [{"name": "Table", "inStock": null}, {"name": "Chair", "inStock": null}]}, 
+    "errors": [
+        {
+            "message": "service 'inventory' response was malformed: graphql response without data must contain at least one error", 
+            "path": ["topProducts", "@"], 
+            "extensions": {"service": "inventory", "reason": "graphql response without data must contain at least one error", "code": "SUBREQUEST_MALFORMED_RESPONSE"}
+        }
+    ]
+}
+```
+
+With this change, the response will look like:
+```json
+{
+    "data": {"topProducts": [{"name": "Table", "inStock": null}, {"name": "Chair", "inStock": null}]},
+    "errors": [
+        {
+            "message": "service 'inventory' response was malformed: graphql response without data must contain at least one error",
+            "path": ["topProducts", 0],
+            "extensions": {"service": "inventory", "reason": "graphql response without data must contain at least one error", "code": "SUBREQUEST_MALFORMED_RESPONSE"}
+        },
+        {
+            "message": "service 'inventory' response was malformed: graphql response without data must contain at least one error",
+            "path": ["topProducts", 1],
+            "extensions": {"service": "inventory", "reason": "graphql response without data must contain at least one error", "code": "SUBREQUEST_MALFORMED_RESPONSE"}
+        }
+    ]
+}
+```
+
+The above examples reflect the behavior with `include_subgraph_errors = true`; if `include_subgraph_errors` is false:
+```json
+{
+    "data": {"topProducts": [{"name": "Table", "inStock": null}, {"name": "Chair", "inStock": null}]},
+    "errors": [
+        {
+            "message": "Subgraph errors redacted",
+            "path": ["topProducts", 0]
+        },
+        {
+            "message": "Subgraph errors redacted",
+            "path": ["topProducts", 1]
+        }
+    ]
+}
+```
+
+By [@carodewig](https://github.com/carodewig) in https://github.com/apollographql/router/pull/7684
+
+### Fix deduplicated subscriptions hanging when one subscription closes ([PR #7879](https://github.com/apollographql/router/pull/7879))
+
+Fixes a regression introduced in v1.50.0. When multiple client subscriptions are deduped onto a single subgraph subscription in WebSocket passthrough mode, and the first client subscription closes, the Router would close the subgraph subscription. The other deduplicated subscriptions would then silently stop receiving events.
+
+Now outgoing subscriptions to subgraphs are kept open as long as _any_ client subscription uses them.
+
+By [@bnjjj](https://github.com/bnjjj) in https://github.com/apollographql/router/pull/7879
+
```

Comment:
I think we should move this up to be next to the other subscriptions fix

Reply:
done. thanks!